cypress/utils/changeEmailTab.js

```diff
@@ -0,0 +1,14 @@
+
+export const  emailTab = ()=>{
+     cy.viewport(1280,720);
+     cy.visit('https://spinkart.neetoauth.net')
```

Comment:
We can avoid visiting base url in every utility fuction if we have already visited the url at the beginning of the test suite.